data/localized/de_patch/text/game/characters.stringtable

```diff
@@ -331,12 +331,12 @@
     </Entry>
     <Entry>
       <ID>103</ID>
-      <DefaultText>Engwithanisches Adra-Bannamulett</DefaultText>
+      <DefaultText>Engwithanisches Adrabannamulett</DefaultText>
```

Comment:
Ich würd die beiden als Übersetzungsfehler ansehen. Es gibt Adra Bán als Edelstein(?)
In der Englischen Version ist es Engwithan Adra Ban Amulet

Aber am verwunderlichsten ist, was das in der charcters Stringtable zu suchen hat.